navigation-animation/src/main/java/com/google/accompanist/navigation/animation/AnimatedComposeNavigator.kt

```diff
@@ -69,14 +71,32 @@ public class AnimatedComposeNavigator : Navigator<AnimatedComposeNavigator.Desti
     /**
      * NavDestination specific to [AnimatedComposeNavigator]
      */
+    @Deprecated(
+        message = "Replace with Androidx ComposeNavigator.Destination and change import to" +
+            "androidx.navigation.compose.ComposeNavigator.",
+        replaceWith = ReplaceWith(
+            "ComposeNavigator.Destination",
+            "androidx.navigation.compose.ComposeNavigator"
+        )
+    )
     @ExperimentalAnimationApi
     @NavDestination.ClassType(Composable::class)
     public class Destination(
         navigator: AnimatedComposeNavigator,
         internal val content: @Composable AnimatedVisibilityScope.(NavBackStackEntry) -> Unit
     ) : NavDestination(navigator)
 
-    internal companion object {
+    companion object {
         internal const val NAME = "animatedComposable"
+
+        @Deprecated(
+            message = "Replace with Androidx ComposeNavigator and change import to" +
```

Comment:
Missing a space at the end of the string

Reply:
Done